pkg/flycheck/role.go

```diff
@@ -27,6 +27,17 @@ func PostgreSQLRole(ctx context.Context, checks *chk.CheckSuite) (*chk.CheckSuit
 	}
 
 	checks.AddCheck("role", func() (string, error) {
+		// checkDisk usage is >90% return "readonly"
+		size, available, err := diskUsage("/data/")
+		if err != nil {
+			return "", errors.Wrap(err, "failed to get disk usage")
```

Comment:
What do you think about making the disk-usage check best-effort?  My thoughts are that if the disk usage check fails, we wouldn't necessarily want to prevent role resolution and break haproxy routing.